osu.Game.Rulesets.Taiko/Edit/Blueprints/HitPlacementBlueprint.cs

```diff
@@ -35,20 +32,8 @@ protected override void LoadComplete()
 
         protected override bool OnMouseDown(MouseDownEvent e)
         {
-            switch (e.Button)
-            {
-                case MouseButton.Left:
-                    HitObject.Type = HitType.Centre;
-                    EndPlacement(true);
-                    return true;
-
-                case MouseButton.Right:
-                    HitObject.Type = HitType.Rim;
-                    EndPlacement(true);
-                    return true;
-            }
-
-            return false;
+            EndPlacement(true);
+            return true;
```

Comment:
I could be okay with this, but one thing - maybe placement should only take place on left mouse button rather than any as this does? On a quick look pretty much all other standard rulesets expect placement to start with left mouse - and constraining to left mouse may avoid confusion from mappers who expect it to work the way it previously did.

Reply:
Yeah, sounds good.